### DISCUSSION TITLE
Show all transactions

### DISCUSSION THREAD

**Discussion by taker218** [firefly-iii/firefly-iii#6645] [General]
Hello,

I don't know if it is possible at the moment, but so far I haven't found the option to do so.
I want to see all transactions (withdrawals/deposits/transfers) from all accounts at once to check if I have missed anything.

At the moment I need to go into every account seperately to check the transactions and it would be easier (at least for me) to have all transactions in one view.

Is this possible in the current layout? Or is that something we need to wait for the new layout?

**Comment by JC5** [comment]
You could use the search. Use `date_after:2020-01-01`.

**Reply by taker218** [reply]
ah that's exactly what I needed. Thanks!
